Xamarin.Essentials/Calendars/Calendars.tizen.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TCalendar = Tizen.Pims.Calendar;
+using TCalendarViews = Tizen.Pims.Calendar.CalendarViews;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Calendars
+    {
+        static TCalendar.CalendarManager manager = new TCalendar.CalendarManager();
+
+        static Task<IEnumerable<Calendar>> PlatformGetCalendarsAsync()
+        {
+            var calendarList = manager.Database.GetAll(TCalendarViews.Book.Uri, 0, 0);
+            return Task.FromResult<IEnumerable<Calendar>>(ToCalendars(calendarList).ToList());
+        }
+
+        static IEnumerable<Calendar> ToCalendars(TCalendar.CalendarList calendarList)
```

Comment:
This is only called after a tolist call, it would be possible to elide the statemachine if we make this method not yield.